src/scripts/charts/AxisChart.js

```diff
@@ -337,7 +337,7 @@ export default class AxisChart extends BaseChart {
 			// let delta = i === 0 ? this.avg_unit_width : x_val - this.x_axis_positions[i-1];
 			if(relX > x_val - this.avg_unit_width/2) {
 				let x = x_val + this.translate_x;
-				let y = this.y_min_tops[i] + this.translate_y;
+				let y = this.y_min_tops[i] + this.translate_y * 2;
```

Comment:
Can you explain the changes to the `AxisChart`? (As it seems the issue was only in Heatmap)

Reply:
Fix changes tooltip positioning from absolute to fixed and this affects all chart types. Changes here are to keep placement correct despite changing coordinates. I'm not sure why multiplication, but it was few lines above and render looked correct.